kingpin/actors/aws/test/integration_ecs.py

```diff
@@ -0,0 +1,195 @@
+"""Simple integration tests for the AWS ECS actors."""
+
+from nose.plugins.attrib import attr
+import logging
+import json
+import os
+import tempfile
+import uuid
+from contextlib import contextmanager
+
+from tornado import testing
+
+from kingpin.actors.aws import ecs
+
+__author__ = 'Steve Mostovoy <smostovoy@nextdoor.com>'
+
+# Generate a common UUID for this particular set of tests
+UUID = uuid.uuid4().hex
+
+log = logging.getLogger(__name__)
+logging.getLogger('boto').setLevel(logging.INFO)
+
+
+class IntegrationECS(testing.AsyncTestCase):
+
+    """High level ECS Actor testing.
+
+    This suite of tests performs the following actions:
+        * Runs a series of successful and failing tasks and wait for completion
+        * Runs a task without waiting
+        * Starts a service with count 0
+        * Changes the scale of service to 1
+        * Changes the scale of service to 5 without waiting
+        * Deletes service
+        * Starts a service with count 1
+        * Changes the scale of service to count 0
+        * Deletes service
+
+    Note, these tests must be run in-order. The order is defined by
+    their definition order in this file. Nose follows this order according
+    to its documentation:
+
+        http://nose.readthedocs.org/en/latest/writing_tests.html
+    """
+
+    integration = True
+
+    region = 'us-west-2'
+    cluster = 'staging-us1'
```

Comment:
So don't hard code our own cluster in here. Make this a generic cluster name, but make it overridable with an environment variable. Same with the region.